modules/transport-netty4/src/test/java/org/elasticsearch/http/netty4/Netty4HttpPipeliningHandlerTests.java

```diff
@@ -232,14 +232,13 @@ public void testSmallFullResponsesAreSentDirectly() {
         assertSame(response, messagesSeen.get(0));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/104728")
     public void testLargeFullResponsesAreSplit() {
         final List<Object> messagesSeen = new ArrayList<>();
         final var embeddedChannel = new EmbeddedChannel(capturingHandler(messagesSeen), getTestHttpHandler());
         embeddedChannel.writeInbound(createHttpRequest("/test"));
         final Netty4HttpRequest request = embeddedChannel.readInbound();
         final var minSize = (int) NettyAllocator.suggestedMaxAllocationSize();
-        final var content = new ZeroBytesReference(between(minSize, minSize * 2));
+        final var content = new ZeroBytesReference(between(minSize, (int) (minSize * 1.5)));
```

Comment:
I chose 1.5 as the upper bound since `testSmallFullResponsesAreSentDirectly` uses 0.5 as the upper bound.